### PR TITLE
Add chown and xattr related operation to jnifuse

### PR DESCRIPTION
**Pull request by maobaolong**

### CODE REVIEW

File: integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java [hunk 1]
```diff
@@ -311,6 +311,23 @@ public int writeCallback(String path, ByteBuffer buf, long size, long offset, By
...
+  }
+
+  public int listxattrCallback(String path, ByteBuffer list, long size) {
+    return 0;
+  }
+
+  public int removexattrCallback(String path, String name) {
+    return 0;
+  }
+
```
Comment (apc999) [inline integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java:329]:
```suggestion
```

File: integration/jnifuse/fs/src/main/java/alluxio/jnifuse/FuseFileSystem.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
 import alluxio.jnifuse.struct.FuseContext;
 import alluxio.jnifuse.struct.FuseFileInfo;
 import alluxio.jnifuse.struct.Statvfs;
+import jnr.ffi.Pointer;
```
Comment (apc999) [inline integration/jnifuse/fs/src/main/java/alluxio/jnifuse/FuseFileSystem.java:18]: wrong import session?
https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#coding-style

File: integration/jnifuse/native/src/main/native/libjnifuse/operation.cc [hunk 3]
```diff
@@ -325,4 +388,51 @@ int RenameOperation::call(const char *oldPath, const char *newPath) {
...
+  this->signature = "(Ljava/lang/String;Ljava/lang/String;)I";
+  this->methodID = env->GetMethodID(this->clazz, "removexattrCallback", signature);
+}
+
+int RemovexattrOperation::call(const char *path, const char *list) {
+  JNIEnv *env = this->fs->getEnv();
+  jstring jspath = env->NewStringUTF(path);
+  jstring jslist = env->NewStringUTF(list);
+
+   int ret = env->CallIntMethod(this->obj, this->methodID, jspath, jslist);
```
Comment (apc999) [inline integration/jnifuse/native/src/main/native/libjnifuse/operation.cc:405]: indent

**Review by apc999 — CHANGES_REQUESTED** [review]
also upload latest .so and .dylib files?

File: integration/jnifuse/native/src/main/native/libjnifuse/operation.cc (hunk 3 above)
Comment (maobaolong) [reply integration/jnifuse/native/src/main/native/libjnifuse/operation.cc:407]: done

File: integration/jnifuse/fs/src/main/java/alluxio/jnifuse/FuseFileSystem.java (hunk 2 above)
Comment (maobaolong) [reply integration/jnifuse/fs/src/main/java/alluxio/jnifuse/FuseFileSystem.java:18]: remove it

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**